indico/modules/events/abstracts/__init__.py

```diff
@@ -134,6 +134,11 @@ def _reviewing_area_visible(conf):
             return False
         return bool(get_user_tracks(conf.as_event, session.user))
 
+    def _submit_abstract_visible(conf):
+        if not session.user or not conf.has_feature('abstracts'):
```

Comment:
I would say no. We had this nested menu items for submitting a registration before and we went for a single page that does everything. Maybe we should do the same here.